ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml

```diff
@@ -1,8 +1,8 @@
 build_root:
   image_stream_tag:
-    name: redhat-appstudio-e2e-test-runner
+    name: redhat-appstudio
     namespace: ci
-    tag: latest
+    tag: e2e-test-runner
```

Comment:
this was an old image used at the time when e2e-tests image was based on go v1.18 and it was not updated after we moved to go v1.19

thus we should be using the image that's being updated by [e2e-tests job](https://github.com/openshift/release/blob/ac05fd6d3bebebff8c8d3e6fefbb881651b0b94d/ci-operator/config/redhat-appstudio/e2e-tests/redhat-appstudio-e2e-tests-main.yaml#L22-L25)